mesoscope/cellpack
# Feature/markdown class

Problem
=======
What is the problem this work solves, including
closes #135 

Solution
========
What I/we did to solve this problem
- created a class `MarkdownWriter` that writes markdown files (incorporating some old code from #163)
- implemented unit tests

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. we are expecting all existing functionality works the same after switching to the `MarkdownWriter`
2. verify that github bot comments in this PR are generated correctly 
3. (optional) run test cases: `pytest cellpack/tests/test_markdown_writer.py`
